web/Web/web/static/js/document-viewer.js

```diff
@@ -255,6 +256,7 @@ docView.prototype = {
       // Show document
       showDocument(docid);
       $(options.docViewSelector).trigger("updated");
+      getScalInfo()
```

Comment:
This shouldn't be called when in using the search interface or any interface other than the discovery interface.

Reply:
You can wrap it around an if 
```
if ($(options. getScalInfoURL) !== null){
    getScalInfo()
}
`